server/datastore/mysql/microsoft_mdm.go

```diff
@@ -940,32 +962,35 @@ func subqueryHostsMDMWindowsOSSettingsStatusVerified() (string, []interface{}, e
 }
 
 func (ds *Datastore) GetMDMWindowsProfilesSummary(ctx context.Context, teamID *uint) (*fleet.MDMProfilesSummary, error) {
-	includeBitLocker, err := ds.GetConfigEnableDiskEncryption(ctx, teamID)
+	diskEncryptionConfig, err := ds.GetConfigEnableDiskEncryption(ctx, teamID)
 	if err != nil {
 		return nil, err
 	}
 
 	var counts []statusCounts
-	if !includeBitLocker {
+	if !diskEncryptionConfig.Enabled {
 		counts, err = getMDMWindowsStatusCountsProfilesOnlyDB(ctx, ds, teamID)
 	} else {
-		counts, err = getMDMWindowsStatusCountsProfilesAndBitLockerDB(ctx, ds, teamID)
+		counts, err = getMDMWindowsStatusCountsProfilesAndBitLockerDB(ctx, ds, teamID, diskEncryptionConfig.BitLockerPINRequired)
 	}
 	if err != nil {
 		return nil, err
 	}
 
 	var res fleet.MDMProfilesSummary
+	// Note that hosts with "BitLocker action required" are counted as pending.
 	for _, c := range counts {
 		switch c.Status {
 		case "failed":
 			res.Failed = c.Count
 		case "pending":
-			res.Pending = c.Count
+			res.Pending += c.Count
 		case "verifying":
 			res.Verifying = c.Count
 		case "verified":
 			res.Verified = c.Count
+		case "action_required":
+			res.Pending += c.Count
```

Comment:
For the profiles summary, count both "pending" and "action_required" BitLocker status as "pending" for the sake of the profile.